python/foglamp/plugins/south/sensehat/sensehat.py

```diff
@@ -256,10 +255,10 @@ def _str_to_bool(s):
                 })
     except RuntimeError as e:
         _LOGGER.exception("Sense HAT runtime error: %s", e)
-        raise exceptions.DataRetrievalError(e)
+        raise e
     except Exception as ex:
         _LOGGER.exception("Sense HAT exception: %s", ex)
-        raise exceptions.DataRetrievalError(ex)
```

Comment:
Please remove DataRetrievalError reference from docstring of plugin_poll. 

and why these two  exception handlers can not be grouped?